modules/tinymce/src/core/main/ts/file/Uploader.ts

```diff
@@ -161,16 +159,17 @@ export function Uploader(uploadStatus: UploadStatus, settings): Uploader {
           resolve(handlerFailure(blobInfo, error, failureOptions));
         };
 
-        progress = (percent) => {
+        progress = (percent: number) => {
           if (percent < 0 || percent > 100) {
             return;
           }
 
-          if (!notification) {
-            notification = openNotification();
-          }
-
-          notification.progressBar.value(percent);
```

Comment:
This was something missed in https://github.com/tinymce/tinymce/pull/6203, as `openNotification` was returning `undefined` since it used `Fun.noop`. As such, this meant that we were getting the following exception:
```
Error: Uncaught TypeError: Cannot read property 'progressBar' of undefined
```